precompiles/slashing/metadata.json

```diff
@@ -0,0 +1,10 @@
Height at which validator was first a candidate OR was unjailed\n    int64 startHeight;\n    /// @dev Index offset into signed block bit array\n    int64 indexOffset;\n    /// @dev Timestamp until which validator is jailed due to liveness downtime\n    int64 jailedUntil;\n    /// @dev Whether or not a validator has been tombstoned (killed out of validator set)\n    bool tombstoned;\n    /// @dev Missed blocks counter (to avoid scanning the array every time)\n    int64 missedBlocksCounter;\n}\n\n/// @dev Params defines the parameters for the slashing module.\nstruct Params {\n    /// @dev SignedBlocksWindow defines how many blocks the validator should have signed\n    int64 signedBlocksWindow;\n    /// @dev MinSignedPerWindow defines the minimum blocks signed per window to avoid slashing\n    Dec minSignedPerWindow;\n    /// @dev DowntimeJailDuration defines how long the validator will be jailed for downtime\n    int64 downtimeJailDuration;\n    /// @dev SlashFractionDoubleSign defines the percentage of slash for double sign\n    Dec slashFractionDoubleSign;\n    /// @dev SlashFractionDowntime defines the percentage of slash for downtime\n    Dec slashFractionDowntime;\n}\n\n/// @author Evmos Team\n/// @title Slashing Precompiled Contract\n/// @dev The interface through which solidity contracts will interact with slashing.\n/// We follow this same interface including four-byte function selectors, in the precompile that\n/// wraps the pallet.\n/// @custom:address 0x0000000000000000000000000000000000000806\ninterface ISlashing {\n    /// @dev Emitted when a validator is unjailed\n    /// @param validator The address of the validator\n    event ValidatorUnjailed(address indexed validator);\n\n    /// @dev GetSigningInfo returns the signing info for a specific validator.\n    /// @param consAddress The validator consensus address\n    /// @return signingInfo The validator signing info\n    function getSigningInfo(\n        address consAddress\n    ) external view returns (SigningInfo memory signingInfo);\n\n    /// @dev GetSigningInfos returns the signing info for all validators.\n    /// @param pagination Pagination configuration for the query\n    /// @return signingInfos The list of validator signing info\n    /// @return pageResponse Pagination information for the response\n    function getSigningInfos(\n        PageRequest calldata pagination\n    ) external view returns (SigningInfo[] memory signingInfos, PageResponse memory pageResponse);\n\n    /// @dev Unjail allows validators to unjail themselves after being jailed for downtime\n    /// @param validatorAddress The validator operator address to unjail\n    /// @return success true if the unjail operation was successful\n    function unjail(address validatorAddress) external returns (bool success);\n\n    /// @dev GetParams returns the slashing module parameters\n    /// @return params The slashing module parameters\n    function getParams() external view returns (Params memory params);\n}\n",
```

Comment:
does source make sense if its just an interface? probably needed for blockscout formatting